### DISCUSSION TITLE
Switch my subscription plan

### DISCUSSION THREAD

**Discussion by mhsheyk** [community/community#118828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there,
I recently upgraded my billing from a monthly to a yearly subscription.
However, I noticed later that my company is trying to get the subscription through our Microsoft professional accounts for all developers.
I want to convert my billing back to monthly and wait for my company to figure this out.
I made a ticket a week ago but am still waiting to hear back. Is there any solution for this?

I appreciate any help you can give me.

**Comment by Akash1134** [comment]
Hi @mhsheyk ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
